frontend/tests/languageSelection.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from "@playwright/test";
+import { navigateTo } from "./common";
+
+test("Language selection", async ({ page }) => {
+    const html = page.locator("html");
+    const languageMenu = page.locator("_react=WithFloatingMenu");
+    const trigger = languageMenu.locator("_react=FloatingTrigger");
```

Comment:
Wouldn't it be better to locate these things via something that is also something a user would use? E.g. `title` attribute must contain "Language selection"?

Reply:
Hm I did that at first but then thought that these selectors maybe shouldn't rely on the translation file. When the translation gets changed at some point, the selector also needs to be changed accordingly or otherwise the test will fail.

On the other hand the `_react` selector doesn't work or rather can't be used on everything, so you'll see me using text content as locator in other places anyway. Meaning we'll need to check the tests anyway when changing translations. 
I'll leave it as is for now, but I'm very open to replacing the `_react` selectors again. They definitely aren't something that makes writing these tests easier (though in some cases they might still be useful, e.g. when an element does not contain any text... whiiich is seldom the case so that point is also kind of moot).

Reply:
I replaced most of the `_react` selectors but some are still required to find elements without any text, such as series blocks. We could also give these a `data-testid` prop but that basicallly does the same as the react selector. Though the react selector is marked as experimental, so it might be useful to swap these anyway.
(Using a test id looks sth like `<SeriesBlock data-testid="seriesBlock">` and then inside our tests use `page.getByTestId("seriesBlock")`, if want you can also take a look at the [docs](https://playwright.dev/docs/locators#locate-by-test-id).)